app/helpers/users_helper.rb

```diff
@@ -4,4 +4,18 @@ def new_or_existing_photo(user)
       user.build_photo
     end
   end
+  
+  def select_options_for_gender
+    [["Male", "M"], ["Female", "F"]]
+  end
+  
+  def gender_select_prompt
```

Comment:
lets not overdo the moving duplicate stuff to helpers path.
keep it linked to options we put in a select box.

Reply:
Done sir.